src/main/java/domain/WinningLotto.java

```diff
@@ -0,0 +1,59 @@
+package domain;
+
+import static util.constant.Message.*;
+import static util.constant.Values.*;
+
+import java.util.*;
+
+public class WinningLotto {
+
+  private List<Integer> winningNumbers;
+  private int bonusNumber;
+
+  public WinningLotto(List<Integer> winningNumbers, int bonusNumber) {
+    validateSize(winningNumbers);
+    validateNumberRange(winningNumbers, bonusNumber);
+    validateDuplicate(winningNumbers, bonusNumber);
+
+    this.winningNumbers = winningNumbers;
+    this.bonusNumber = bonusNumber;
+  }
```

Comment:
유효성 검증을 꼼꼼하게 해주신덕에 주생성자 로직이 꽤 길어졌군요? 😃
아래와 같이 유효성 검증 로직들을 하나의 메서드로 묶어주는 것도 가독성에 도움이 될듯 합니다. 😉 

```suggestion
  public WinningLotto(List<Integer> winningNumbers, int bonusNumber) {
      validateWinningLotto(winningNumbers, bonusNumber)
      this.winningNumbers = winningNumbers;
      this.bonusNumber = bonusNumber;
  }
  
  private void validateWinningLotto(List<Integer> winningNumbers, int bonusNumber) {
      validateSize(winningNumbers);
      validateNumberRange(winningNumbers, bonusNumber);
      validateDuplicate(winningNumbers, bonusNumber);  
  }
```